### DISCUSSION TITLE
Evaluation hardware

### DISCUSSION THREAD

**Discussion by kschwan** [surgical-robotics-ai/surgical_robotics_challenge#45] [Q&A] [answered]
Hi,

I'd like to ask on what hardware you will run the evaluation? In particular, is CUDA(-enabled hardware) available, or should we assume everything runs on CPU only? I ask because there's a risk we could overstep the 10 sec limit in Challenge 1. We'd have to test this at least 🙂

**Comment by pkazanzides** [comment]
This is a good question.  When we initially launched the challenge, it was not clear how well CUDA was supported in Docker containers. I now see more recent posts, such as [this one](https://www.cloudsavvyit.com/14942/how-to-use-an-nvidia-gpu-with-docker-containers/), that explain how to do it.  I am not a Docker expert, but it seems that we would need to make some changes on the host PC and install some packages in the Docker container.  The Docker containers that we provided do not have these packages, but I have no objection to adding them and then using a host PC with a decent GPU for the evaluation.  If you, or anyone else, has already configured your system to use a GPU, we would appreciate information about how you did that and even a pull request to update our docker files.

**Comment by kschwan** [comment]
Hi Peter,
I'm also not a Docker expert. I hadn't considered that Docker and GPU access might be an issue. It looks like it should be possible though. Before looking into it further, I'll test how well our implementation runs on CPU only.

**Comment by pkazanzides** [comment, marked as answer]
We just updated the website to change the Challenge 1 time limit from 10 seconds to 60 seconds, which should eliminate concerns about the evaluation hardware.

**Reply by kschwan** [reply]
OK, sounds good 😊 Then we should be fine either way
